app/services/dyes_service.py

```
@@ -0,0 +1,100 @@
+        self.gw2_client = gw2_client
+
+    async def get_all_dyes(self) -> list[dict]:
+        """
+        Get all dyes from GW2 API and sync with database in the background.
+        If API fails, fallback to database.
+        """
+        try:
+            dyes_data = await self._get_dyes_from_api()
+
+            # Sync with db in background
+            asyncio.create_task(self._sync_dyes_to_db(dyes_data))
```

Ruff (0.14.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)